demos/serverless/deploy.js

```diff
@@ -12,6 +12,10 @@ let useEventBridge = false;
 let enableTerminationProtection = false;
 let disablePrintingLogs = false;
 let chimeEndpoint = 'https://service.chime.aws.amazon.com';
+let chimeMeetingsIADEndpoint = '-';
+let chimeMeetingsPDXEndpoint = '-';
+let chimeMeetingsFRAEndpoint = '-';
+let chimeMeetingsSINEndpoint = '-';
```

Comment:
Should these defaults be set to their actual values?

Reply:
As this code is public github repository we cannot put the actual values.

Reply:
We can't put the public prod endpoints here?

Reply:
Actually we do not want the default values to be prod values, As in case when we provide endpoint = gammaEndpoint and we do not provide the regional endpoints it will use prod regional endpoints which would be incorrect. I have put in a condition in handlers.js at line 44 `const regionalEndpoint = (chimeMeetingsRegionalEndPoints[controlRegion] === '-') ? endpoint : chimeMeetingsRegionalEndPoints[controlRegion];`  so in case if regional endpoint is '-' we would use endpoint provided to make it backward compatible.